src/app/modules/shared_models/message_item.nim

```diff
@@ -87,6 +90,7 @@ proc initItem*(
     sticker: string,
     stickerPack: int,
     links: seq[string],
```

Comment:
Is `links` still used?

Reply:
Well, it's still part of the Message in status-go. Don't now if it makes sense to drop it now.